prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html

```diff
@@ -267,17 +267,17 @@
     <ng-container *ngFor="let unlistedCertification of unlistedCertifications; let i = index;">
 
       <app-enrollee-property title="College Name"
-        [makeBold]="true">
+                             [makeBold]="true">
         {{ unlistedCertification.collegeName | default }}
       </app-enrollee-property>
 
       <app-enrollee-property title="College Licence"
-        [makeBold]="true">
+                             [makeBold]="true">
         {{ unlistedCertification.licenceNumber | default }}
       </app-enrollee-property>
 
       <app-enrollee-property title="Renewal Date"
-        [makeBold]="true">
+                             [makeBold]="true">
```

Comment:
added unnecessary indentation

Reply:
Tried to remove the additional indentation but VS code just keeps adding them back upon save. It seems VS code is trying to match the title attribute, just like other indentation lower below. Since it is just formatting issue, I will left it as it is now.